src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -77,35 +75,13 @@ type CA struct {
 	Cert       string // PEM-encoded
 }
 
-func exists(file string) (os.FileInfo, bool) {
-	if fi, err := os.Stat(file); os.IsNotExist(err) {
-		log.Debugf("File does not exist at %v", file)
-		return fi, false
-	} else {
-		log.Debugf("File exists at %v", file)
-		return fi, true
+// hasCustomChainedServer returns an error if the config doesn't have a custom
+// chained server.
+func hasCustomChainedServer(_cfg yamlconf.Config) error {
+	cfg, ok := _cfg.(*Config)
```

Comment:
We are assuming here that just casting to a Config will provide a chained server. The case where we have a valid cast without a valid chained server is not going to happen?